libraries/AP_BattMonitor/AP_BattMonitor_Analog.cpp

```diff
@@ -26,6 +26,11 @@ AP_BattMonitor_Analog::read()
 
     // get voltage
     _state.voltage = _volt_pin_analog_source->voltage_average() * _mon._volt_multiplier[_state.instance];
+    
+    // set watt capacity
+    if (_state.watts_capacity == 0 && _state.voltage_resting_estimate != 0 && _mon._pack_capacity != 0) {
+            _state.watts_capacity = _state.voltage_resting_estimate * (float)((uint32_t)_mon._pack_capacity / 1000);
```

Comment:
@Pedals2Paddles I think this is not the correct way to calculate Wh capacity.
Generally Wh value could be calculated as [capacity in Ah] * [average pack voltage]. Where average pack voltage is a constant, that is varies for different battery type (3.7 for 1S lipo).
But, even the approach above will not provide good accuracy. The only robust way is to have explicit setting watts_capacity. 
I mean user have to fly-out a battery then check his log for watts_used value. This will be the real value and this value need to set to watts_capacity.